### DISCUSSION TITLE
Question about expanding distributed minio setup

### DISCUSSION THREAD

**Discussion by AlexZIX** [minio/minio#19882] [Q&A] [answered]
Hi,

the time of expanding my existing distibuted minio setup is approaching and after reading docs how to do that I still have few questions:

1. Is it possible to add new erasure set of new disks on existing nodes, or I must create new nodes anyway.

For now my cluster starting with this MINIO_VOLUMES option:

http://minio-cold-{1...4}.bazissoft.ru/hdd-pools/hdd-pool-{1...4}

Can I do that:

http://minio-cold-{1...4}.bazissoft.ru/hdd-pools/hdd-pool-{1...4} http://minio-cold-{1...4}.bazissoft.ru/hdd-pools/hdd-pool-{5...8}

Or I must do it only this way:

http://minio-cold-{1...4}.bazissoft.ru/hdd-pools/hdd-pool-{1...4} http://minio-cold-{5...8}.bazissoft.ru/hdd-pools/hdd-pool-{1...4}

2. Can I rename the fqdn from minio-cold-{1...4}.bazissoft.ru to minio-{1...4}.bazissoft.ru? 

3. Docs said that minio nodes should be restarted simultaneously during expansion, not rolling, but it's not too easy. Can I stop all existing nodes and then start it one-by-one and then start nodes from the new pool?

**Comment by klauspost** [comment, marked as answer]
1. No
2. I believe so, but don't remember the details.
3. Yes.

**Comment by AlexZIX** [comment]
Thanks
